assets/js/modules/analytics/datastore/accounts.test.js

```diff
@@ -36,8 +36,15 @@ import * as fixtures from './__fixtures__';
 
 describe( 'modules/analytics accounts', () => {
 	let apiFetchSpy;
+	let windowLocationSpy;
 	let registry;
 	let store;
+	let redirect;
+
+	const accountName = 'Test Account';
+	const propertyName = 'Test Property';
+	const profileName = 'Test Profile';
```

Comment:
Let's take these three values from `fixtures.accountTicket` instead, they're duplicated here, and it's not clear that they should be equal because of that. Also, let's define these variables in each test where they're needed, it makes the individual tests easier to understand just by looking at the test code itself.

Reply:
👍